docs/io/configuration/components/models/index.rst

```diff
@@ -0,0 +1,172 @@
+.. _model-csvy-and-config:
+
+******
+Models
+******
+
+TARDIS requires a model of the homologously expanding ejecta in order to run a simulation (see :ref:`model`).
+A model will include information like the velocity shell structure, abundances, density, etc.
+TARDIS offers two ways of specifying the model: either directly in the configuration YAML file
+or separately in a model.csvy file. See `here <https://csvy.org/>`_ for an explanation of
+the CSVY file format and :ref:`here <config-file>` for a link to the csvy_model schema.
+
+.. note::
+    We highly recommend using the cleaner CSVY format.
```

Comment:
Not sure this is true in general, but rather for specific custom density/abundance models

Reply:
What should I put there? This was copied and pasted directly from the old documentation